db/tests.robot

```diff
@@ -1,15 +1,152 @@
     Table Must Exist    tsao_accounts
+    [Teardown]    Disconnect From Database
+
+Insert Data into tsao_account Table
+    [Documentation]    Test to INSERT data into tsao_accounts table.
+    [Setup]    Connect to DB
+    ${output} =    Execute SQL String    INSERT INTO tsao_accounts (ID, Name, Username, Password, Role, CreationDate, IsApproved, IsDeleted) VALUES (6, 'Steve Jobs', 'sj1955', 'f223faa96f22916294922b171a2696d868fd1f9129302eb41a45b2a2ea2ebbfd', 'User', '2024-01-22 12:34:56', true, false);
+    Log    ${output}
+    Should Be Equal As Strings    ${output}    None
+    [Teardown]    Disconnect From Database
+
+Update Data in tsao_account Table
+    [Documentation]    Test to Update data into tsao_accounts table.
+    [Setup]    Connect to DB
+    ${output} =    Execute SQL String    UPDATE tsao_accounts SET Username = 'sj55', Role= 'Administrator' WHERE ID = 6;   
+    Log    ${output}
+    Should Be Equal As Strings    ${output}    None
+    [Teardown]    Disconnect From Database
+
+Delete Data in tsao_account Table
+    [Documentation]    Test to Delete data into tsao_accounts table.
+    [Setup]    Connect to DB
+    ${output} =    Execute SQL String    DELETE FROM tsao_accounts WHERE ID=6;   
+    Log    ${output}
+    Should Be Equal As Strings    ${output}    None
+    [Teardown]    Disconnect From Database
+
+Verify Columns Existence
+    [Documentation]    Verify that all columns exist in the 'tsao_accounts' table.
+    [Setup]    Connect to DB
+    ${columns}    Query    SHOW COLUMNS FROM tsao_accounts
+    @{column_names}=    Create List
+    FOR    ${column}    IN    @{columns}
+        Append To List    ${column_names}    ${column[0]}  # Use index 0 to access the 'Field' value
+    END
+    List Should Contain Value    ${column_names}    ID
+    List Should Contain Value    ${column_names}    Name
+    List Should Contain Value    ${column_names}    Username
+    List Should Contain Value    ${column_names}    Password
+    List Should Contain Value    ${column_names}    Role
+    List Should Contain Value    ${column_names}    CreationDate
+    List Should Contain Value    ${column_names}    IsApproved
+    List Should Contain Value    ${column_names}    IsDeleted
+    [Teardown]    Disconnect From Database
+Verify PrimaryKey
+    [Documentation]    Verify the primary key and auto-increment for the 'ID' column.
+    [Setup]    Connect to DB
+    ${id_attributes}    Query    SHOW COLUMNS FROM tsao_accounts WHERE Field='ID'
+    ${attribute_info}    Set Variable    ${id_attributes[0]}
+    ${attribute_list}    Create List    ${attribute_info[3]}
+    List Should Contain Value    ${attribute_list}    PRI
+    ${auto_increment}    Set Variable If    '${attribute_info[5]}' != 'NO'    ${attribute_info[5]}    NOT FOUND
+    Should Not Be Equal As Strings    ${auto_increment}    NOT FOUND
+    Should Not Be Equal As Strings    ${auto_increment}    NO
+    [Teardown]    Disconnect From Database
+
+Verify ColumnDataTypes
```

Comment:
Which table?

Reply:
Also let's try to make it as expressive as possible so instead of ColumnDataTypes, use Column Data Types, like how it's normally written